docs/openapi/organizations-api.yaml

```diff
@@ -0,0 +1,47 @@
+organizations:
+  post:
+    tags:
+      - organization
+    summary: Create a new organization
+    description: |
+      This endpoint is useful for creating a new organization.
+    operationId: createOrganization
+    requestBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: './schemas.yaml#/OrganizationCreate'
+    responses:
```

Comment:
Is `400 Bad Request` a possibility here, too?

Reply:
not currently, as our error handling in the controller is not up to snuff yet. should definitely be added at some point. if a site creation is missing required properties it would currently result in a 500 error.

Reply:
@blefebvre created https://github.com/adobe/spacecat-api-service/issues/114 to track this